src/main/java/com/mkul/game/ox/Players.java

```diff
@@ -0,0 +1,56 @@
+package com.mkul.game.ox;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+class Players {
+  final Queue<Player> players;
+
+  private Players(Player player1, Player player2) {
+    players = new LinkedList<>(List.of(player1, player2));
+  }
+
+  Player getCurrent() {
+    return players.peek();
+  }
+
+  Player getNext() {
+    Player player = players.poll();
+    players.add(player);
+    return players.peek();
+  }
```

Comment:
Method name does not tell what it really does ((poll -> add) -> peek). In the previous method there was also "get" prefix but method did only peek.